### ISSUE TITLE
[New Feature] Allow hotkey bind to mouse buttons.

### ISSUE THREAD

**Issue by Lylaktech** [Vinifera-Developers/Vinifera#51]
### Description:
Currently the game does not allow you to bind hotkeys to mouse buttons like middle mouse (click) and other extra buttons it might have, would be neat to bind mouse dependent hotkeys like Q-move to a button on the mouse, becomes intuitive to use.

**Comment by CCHyper** [comment]
I don't think it will be possible to rebind left, right or middle mouse buttons, but maybe possible to support extended mouse buttons. The issue there might be that there is no standard on these?
